README.md

```diff
@@ -152,5 +152,6 @@ Cerberus Management Service is released under the [Apache License, Version 2.0](
 [license]:LICENSE.txt
 [license img]:https://img.shields.io/badge/License-Apache%202-blue.svg
 
+
 [coveralls]:https://coveralls.io/github/Nike-Inc/cerberus
```

Comment:
Because we are no longer using Coveralls, I believe you can delete these two lines.